chia/util/initial-config.yaml

```diff
@@ -73,13 +73,10 @@ network_overrides: &network_overrides
       GENESIS_PRE_FARM_POOL_PUZZLE_HASH: d23da14695a188ae5708dd152263c4db883eb27edeb936178d4d988b8f3ce5fc
       MEMPOOL_BLOCK_BUFFER: 10
       MIN_PLOT_SIZE: 18
-      SOFT_FORK2_HEIGHT: 3000000
```

Comment:
If we do not support testnet10 then maybe we should just remove the entire config?

Reply:
I don't know much about this, but a thought is that perhaps we do still work with these other testnets?  Whereas here we are making it so we explicitly will not work with tn10.  I don't know if that's an important distinction, or a real one, or not.

Is there a reason that testnet0 is left?

Reply:
it seems that testnet0 is what we use in the simulator